sources/ElkArte/Http/BrowserDetector.php

```diff
@@ -472,7 +472,7 @@ public function isPossibleRobot()
 		// Be you robot or human?
 		if (!isset($this->_browsers['possibly_robot']))
 		{
-			if (User::$info->possibly_robot)
+			if (isset(User::$info->possibly_robot))
```

Comment:
Just wondering: is this php8 giving errors again?

Reply:
I don't remember if this was 7.4 / 8 issue, but for sure if things don't 100% pass back a bool then compares break.  If I either saw it while running or in the error log it got "patched"

Reply:
Good to know!